biz.aQute.repository/src/aQute/bnd/repository/p2/provider/ArtifactID.java

```diff
@@ -0,0 +1,51 @@
+package aQute.bnd.repository.p2.provider;
+
+class ArtifactID {
+	private final String	id;
+	private final String	version;
```

Comment:
Two different kinds of `Version`. Figured it was just a tad simpler than converting between `aQute.bnd.version.Version` and `org.osgi.framework.Version`.